openwisp_monitoring/device/migrations/0002_create_template.py

```diff
@@ -0,0 +1,103 @@
         type = 'wireless',\n                wireless = {\n                    ssid = iwinfo.ssid,\n                    mode = iwinfo_modes[iwinfo.mode] or iwinfo.mode,\n                    channel = iwinfo.channel,\n                    frequency = iwinfo.frequency,\n                    tx_power = iwinfo.txpower,\n                    signal = iwinfo.signal,\n                    noise = iwinfo.noise,\n                    country = iwinfo.country\n                }\n            }\n            if clients and next(clients.clients) ~= nil then\n                netjson_interface.wireless.clients = netjson_clients(clients.clients)\n            end\n            wireless_interfaces[name] = netjson_interface\n        end\n    end\nend\n\n-- collect interface stats\nfor name, interface in pairs(network_status) do\n    -- only collect data from iterfaces which have not been excluded\n    if not is_excluded(name) then\n        netjson_interface = {\n            name = name,\n            type = string.lower(interface.type),\n            up = interface.up,\n            mac = interface.macaddr,\n            txqueuelen = interface.txqueuelen,\n            mtu = interface.mtu,\n            speed = interface.speed,\n            bridge_members = interface['bridge-members'],\n            multicast = interface.multicast\n        }\n        if wireless_interfaces[name] then\n            dict_merge(wireless_interfaces[name], netjson_interface)\n            interface.type = netjson_interface.type\n        end\n        if interface.type == 'Network device' then\n            link_supported = interface['link-supported']\n            if link_supported and next(link_supported) then\n                netjson_interface.type = 'ethernet'\n                netjson_interface.link_supported = link_supported\n            elseif vpn_interfaces[name] then\n                netjson_interface.type = 'virtual'\n            else\n                netjson_interface.type = 'other'\n            end\n        end\n        if include_stats[name] then\n            netjson_interface.statistics = interface.statistics\n        end\n        addresses = get_addresses(name)\n        if next(addresses) then\n            netjson_interface.addresses = addresses\n        end\n        info = get_interface_info(name, netjson_interface)\n        if info.stp ~= nil then\n            netjson_interface.stp = info.stp\n        end\n        table.insert(interfaces, netjson_interface)\n        -- DNS info is independent from interface\n        if info.dns_servers then\n            array_concat(info.dns_servers, dns_servers)\n        end\n        if info.dns_search then\n            array_concat(info.dns_search, dns_search)\n        end\n    end\nend\n\nif next(interfaces) ~= nil then\n    netjson.interfaces = interfaces\nend\nif next(dns_servers) ~= nil then\n    netjson.dns_servers = dns_servers\nend\nif next(dns_search) ~= nil then\n    netjson.dns_search = dns_search\nend\n\nprint(cjson.encode(netjson))\n",  # noqa
```

Comment:
please update this with the contents from this file: https://github.com/openwisp/lua-monitoring/blob/master/netjson-monitoring.lua
I found out this version I passed you was not up to date.